helm/hpcc/templates/network.yaml

```diff
@@ -23,6 +23,7 @@
 */}}
 {{- $egress := (.Values.global.egress | default dict) -}}
 {{- $egressRestricted := $egress.restricted | default false -}}
+{{- $metrics := .Values.global.metrics }}
```

Comment:
wondering how this works if individual components have overriden some settings, resulting in different metrics per component (i.e. the result of generateMetricsConfig) ?